superset-frontend/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -138,6 +139,7 @@ class HeaderActionsDropdown extends React.PureComponent {
       isLoading,
       refreshLimit,
       refreshWarning,
+      lastModifiedTime,
```

Comment:
This thing is getting a little too bloated. I'm wondering whether we can refactor and just pass a `dashboard` object to this component.

Reply:
yes. `dashboardInfo` is read only dashboard metadata. I will consider group other edit-able dashboard metadata into another object.